examples/deadline/Local-Zone/ts/lib/service-tier.ts

```diff
@@ -0,0 +1,162 @@
+   * @default The latest available release of Deadline is used
+   */
+  readonly deadlineVersion?: string;
+}
+
+/**
+ * The service tier contains all "business-logic" constructs (e.g. Repository, Render Queue, etc.).
+ */
+export class ServiceTier extends Stack {
+  /**
+   * The render queue.
+   */
+  public readonly renderQueue: RenderQueue;
+
+  /**
+   * The version of Deadline configured by the app.
+   */
+  public readonly version: VersionQuery;
+
+  /**
+   * Initializes a new instance of {@link ServiceTier}.
+   */
+  constructor(scope: Construct, id: string, props: ServiceTierProps) {
+    super(scope, id, props);
+
+    this.version = new VersionQuery(this, 'Version', {
+      version: props.deadlineVersion,
+    });
+
+    // We are excluding the local zones from the Repository. This construct will create an
+    // EFS filesystem and DocDB cluster, both of which aren't available in any local zones at this time.
+    const repositorySubnets: SubnetSelection = {
+      availabilityZones: [ props.availabilityZones[0] ],
+      subnetType: SubnetType.PRIVATE,
+    };
+    const repository = new Repository(this, 'Repository', {
+      vpc: props.vpc,
+      version: this.version,
+      removalPolicy: {
+        database: RemovalPolicy.DESTROY,
+        filesystem: RemovalPolicy.DESTROY,
+      },
+      repositoryInstallationTimeout: Duration.minutes(20),
+      vpcSubnets: repositorySubnets,
+    });
+
+    const images = new ThinkboxDockerImages(this, 'Images', {
+      version: this.version,
+      userAwsThinkboxEulaAcceptance: props.acceptAwsThinkboxEula,
+    });
+
+    const serverCert = new X509CertificatePem(this, 'RQCert', {
+      subject: {
+        cn: `renderqueue.${props.dnsZone.zoneName}`,
+        o: 'RFDK-Sample',
+        ou: 'RenderQueueExternal',
+      },
+      signingCertificate: props.rootCa,
+    });
+
+    // The render queue is also put only in the standard availability zones. The service itself
+    // is run in a single zone, while the load balancer that sits in front of it can be provided
+    // all the standard zones we're using.
+    const renderQueueSubnets: SubnetSelection = {
+      availabilityZones: [ props.availabilityZones[0] ],
+      subnetType: SubnetType.PRIVATE,
+    };
+    const renderQueueAlbSubnets: SubnetSelection = {
+      availabilityZones: props.availabilityZones,
+      subnetType: SubnetType.PRIVATE,
+      onePerAz: true,
+    };
+    this.renderQueue = new RenderQueue(this, 'RenderQueue', {
+      vpc: props.vpc,
+      images: images,
+      repository,
+      hostname: {
+        hostname: 'renderqueue',
+        zone: props.dnsZone,
+      },
+      trafficEncryption: {
+        externalTLS: {
+          rfdkCertificate: serverCert,
+        },
+        internalProtocol: ApplicationProtocol.HTTPS,
+      },
+      version: this.version,
+      vpcSubnets: renderQueueSubnets,
+      vpcSubnetsAlb: renderQueueAlbSubnets,
+      deletionProtection: false,
```

Comment:
Same comment as the python app's Render Queue about deletion protection, should we consider adding the `TODO` comment here?